dataline-config/init/src/main/resources/config/DESTINATION_CONNECTION_SPECIFICATION/8442ee76-cc1d-419a-bd8b-859a090366d4.json

```diff
@@ -0,0 +1,27 @@
+{
+  "destinationId": "8be1cf83-fde1-477f-a4ad-318d23c9f3c6",
+  "destinationSpecificationId": "8442ee76-cc1d-419a-bd8b-859a090366d4",
+  "specification": {
+    "$schema": "http://json-schema.org/draft-07/schema#",
+    "title": "Singer CSV Target Spec",
+    "type": "object",
+    "required": [],
+    "additionalProperties": false,
+    "properties": {
+      "delimiter": {
+        "description": "Delimiter used to separate fields.",
+        "type": "string",
+        "examples": [","]
+      },
+      "quotechar": {
+        "description": "What character in the input should be treated as a quotation mark.",
```

Comment:
I think people will be confused by what this means. "Input" is a little ambiguous and it isn't clear how this manifests itself. Does this mean these characters will be escaped?

Reply:
What about:
Character used to quote strings in the CSV